### ISSUE TITLE
https://github.com/michaelWuensch/BitBanana/commit/30acfc3b72768ef3a46f5aa6561bf2e91328ef8c

### ISSUE THREAD

**Issue by Pit09123** [michaelWuensch/BitBanana#74]

**Comment by michaelWuensch** [comment]
For your interest, here is an explanation for the License changes:
- I programmed Zap Android and agreed with Jack Mallers that his name will be in the License as Copyright.
- I stopped working on Zap, forked the project and renamed the fork to BitBanana. But when renaming it, I needed to convert the fork into an own Github Project, basically deforking it. (Forks are not crawled and you wouldn't be able to google for "BitBanana github" and find it.) Because of that process it got some default License.
- I changed it to MIT with my name as it has no longer any affiliation with Zap or Jack Mallers.
- I realized I cannot just change the name as the code was released with the copyright to Jack Mallers, that's why I reverted back to Jack Mallers in the commit you specified.
- After I added new code I added my name to the copyright.
- Because of the history I think I cannot remove Jack Mallers from the license even though he hasn't written one line of code.
